draft-ietf-jose-hpke-encrypt.md

```diff
@@ -196,7 +194,15 @@ In HPKE JWE Integrated Encryption:
 - The "encrypted_key" MUST be the base64url encoded encapsulated key as defined in Section 5.1.1 of {{RFC9180}}.
 - The "iv", "tag" and "aad" members MUST NOT be present.
 - The "ciphertext" MUST be the base64url encoded ciphertext as defined in Section 5.2 of {{RFC9180}}.
-- The HPKE Setup info parameter MAY be used, and its values are not constrained by this specification. By default, it is empty unless apu or apv are present, in which case it will carry the JOSE context-specific data as defined in Section 4.6.2 of {{RFC7518}}, i.e., the concatenation of AlgorithmID, PartyUInfo, and PartyVInfo. It does not include Z, keydatalen, SuppPubInfo, or SuppPrivInfo. AlgorithmID is structured as defined in Section 4.6.2 of {{RFC7518}} and the Data field in AlgorithmID will be set to the octets of the ASCII representation of the "enc" Header Parameter value.
+- The HPKE Setup info parameter MAY be used, and its values are not constrained by this specification.
+- The HPKE AEAD AAD MUST be set to the "JWE Additional Authenticated Data encryption parameter" defined in Step 14 of Section 5.1 of {{RFC7516}}.
+
+
+If the "zip" header parameter is present, the resulting plaintext is uncompressed using the algorithm specified and the result is the
+raw message plaintext.
+
```

Comment:
```suggestion
```

Talking about zip is unnecessary here because JWE specifies this behavior, which applies to all algorithms.

Reply:
The text is only specified for the new JWE Integrated Encryption mode.

Reply:
A reference would be better than repeating the text... some thing like:

Not that compression is possible with both integrated encryption and key encryption, see section .... of rfc ...

Reply:
I don't see a need to re-iterate that compression is possible with key encryption but I agree to avoid repeating the text.